src/core/tests/library/LegacyEnvLayerExtensions.py

```diff
@@ -226,6 +226,53 @@ def run_command_output(self, cmd, no_output=False, chk_err=True):
+                                 "Retrieving repository 'SLE-Module-CAP-Tools15-SP3-Pool' metadata .................................................................[done]\n" + \
+                                 "Building repository 'SLE-Module-CAP-Tools15-SP3-Pool' cache ......................................................................[done]\n" + \
+                                 "Retrieving repository 'SLE-Module-CAP-Tools15-SP3-Updates' metadata ..............................................................[done]\n" + \
+                                 "Building repository 'SLE-Module-CAP-Tools15-SP3-Updates' cache ...................................................................[done]\n" + \
+                                 "Retrieving repository 'SLE-Product-SLES15-SP3-Pool' metadata .....................................................................[done]\n" + \
+                                 "Building repository 'SLE-Product-SLES15-SP3-Pool' cache ..........................................................................[done]\n" + \
+                                 "Retrieving repository 'SLE-Product-SLES15-SP3-Updates' metadata ..................................................................[done]\n" + \
+                                 "Building repository 'SLE-Product-SLES15-SP3-Updates' cache .......................................................................[done]\n" + \
+                                 "Retrieving repository 'SLE-Module-Server-Applications15-SP3-Pool' metadata .......................................................[done]\n" + \
+                                 "Building repository 'SLE-Module-Server-Applications15-SP3-Pool' cache ............................................................[done]\n" + \
+                                 "Retrieving repository 'SLE-Module-Server-Applications15-SP3-Updates' metadata ....................................................[done]\n" + \
+                                 "Building repository 'SLE-Module-Server-Applications15-SP3-Updates' cache .........................................................[done]\n" + \
+                                 "Retrieving repository 'SLE-Module-Web-Scripting15-SP3-Pool' metadata .............................................................[done]\n" + \
+                                 "Building repository 'SLE-Module-Web-Scripting15-SP3-Pool' cache ..................................................................[done]\n" + \
+                                 "Retrieving repository 'SLE-Module-Web-Scripting15-SP3-Updates' metadata ..........................................................[done]\n" + \
+                                 "Building repository 'SLE-Module-Web-Scripting15-SP3-Updates' cache ...............................................................[done]\n" + \
+                                 "All repositories have been refreshed."
```

Comment:
Trim it to the minimum required for the test to work.

Reply:
Done